mzLib/Readers/QuantificationResults/QuantifiedPeak.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CsvHelper.Configuration;
+using CsvHelper.Configuration.Attributes;
+
+namespace Readers.QuantificationResults
+{
+    public class QuantifiedPeak
+    {
+        public static CsvConfiguration CsvConfiguration = new CsvConfiguration(System.Globalization.CultureInfo.InvariantCulture)
+        {
+            Delimiter = "\t",
+            HasHeaderRecord = true,
+            IgnoreBlankLines = true,
+            TrimOptions = CsvHelper.Configuration.TrimOptions.Trim,
+        };
+
+        [Name("File Name")]
+        public string FileName { get; set; }
+
+        [Name("Base Sequence")]
+        public string BaseSequence { get; set; }
+
+        [Name("Full Sequence")]
+        public string FullSequence { get; set; }
+
+        [Name("Protein Group")]
+        public string ProteinGroup { get; set; }
+
+        [Name("Peptide Monoisotopic Mass")]
+        public double PeptideMonoisotopicMass { get; set; }
+
+        [Name("MS2 Retention Time")]
+        public double MS2RetentionTime { get; set; }
```

Comment:
This should be nullable, otherwise MBR peaks will throw an exception

Reply:
fixed this